<?xml version="1.0"?>

<launch>
  <!-- Start the laser_scan_merger node -->
  <node name="auto_trax_laser_scan_merger_node" pkg="auto_trax_sensors" type="laser_scan_merger_node" output="screen">
    <param name="merged_scan_pub_topic" value="merged_scan" />
    <param name="scan_sub_topic" value="/scan" />
    <param name="frame_id_left" value="camera_left" />
    <param name="frame_id_right" value="camera_right" />
    <param name="angle_increment" value="0.01" />
    <param name="left_camera_offset" value="0.15" />
    <param name="left_camera_orientation" value="45" />
    <param name="right_camera_offset" value="-0.15" />
    <param name="right_camera_orientation" value="-45" />
  </node>
</launch>
